include/misc/rb.h

```diff
@@ -112,7 +112,7 @@ static inline struct rbnode *rb_get_max(struct rbtree *tree)
  * implement a "set" construct by simply testing the pointer value
  * itself.
  */
-int rb_contains(struct rbtree *tree, struct rbnode *node);
+bool rb_contains(struct rbtree *tree, struct rbnode *node);
```

Comment:
Out of curiosity, which MISRA rule is this?

Reply:
MISRA 10.5

Reply:
> MISRA 10.5

please mention those in the commit messages.

Reply:
yep, I've asked it in other commit too.

Reply:
>please mention those in the commit messages.

This is quite hard to do because I fixed various issues, i cant pin point which specific rule was tackled. This is because I am moving one file at a time.